src/isIsogram.test.js

```diff
@@ -6,4 +6,31 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`True with 'isog'`, () => {
+    expect(isIsogram('isog'))
+      .toBeTruthy();
+  });
+
+  it(`False with 'isogg'`, () => {
+    expect(isIsogram('isogg'))
+      .toBeFalsy();
+  });
+
+  it(`takes only letters'`, () => {
```

Comment:
Consider providing more descriptive test case names. For example, instead of `takes only letters`, you could use `throws an error when input is not a string`.